.github/workflows/preview-comment.yml

```diff
@@ -54,18 +50,7 @@ jobs:
               })
               throw new Error('not allowed')
             }
-
-      - name: Checkout code
-        uses: actions/checkout@v4
-
-      - name: Install pnpm
-        uses: pnpm/action-setup@v4.0.0
-
-      - name: Install dependencies
-        run: pnpm install
-
-      - name: Build
-        working-directory: ./packages/vite
-        run: pnpm build
-
-      - run: pnpm dlx pkg-pr-new@0.0 publish --compact --pnpm ./packages/vite
+      - run: |
+          gh issue edit ${{ github.event.issue.number }} --add-label "pr: preview" --repo ${{ github.repository }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Might this have an issue with the default token limitation @userquin was mentioning in https://github.com/vitest-dev/vitest/pull/6362#issuecomment-2376598561?

> Actually, github actions can also add labels but labels added via their default bot cannot trigger other actions. 🫠 That's why add it using separate user.

Reply:
Hmm, interesting, didn't know that! isn't there any workaround to this? 

because adding custom tokens might not feel that good!

Reply:
If there's no solution, maybe we can go with the manual label addition! 

cc @patak-dev

Reply:
Ya, I think the Vitest setup may work well for Vite too.

Reply:
You can use a bot, check vitepress, ask @brc-dd

Reply:
just made the change! 

"trigger: preview" would cause a publish. Then that label would be removed for next time runs, so they'd be caused by adding the same label again.